app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -143,7 +143,7 @@ class MainActivity : ComponentActivity() {
                 )
                 Button(
                     onClick = {
-                        var incidentInfo = Incident().apply {
+                        var incidentInfo = Incident("1").apply {
```

Comment:
What's the meaning of the "1" here?  It looks like a magic number.  Make a constant from this, and give the constant a descriptive name, so that the code is self-describing.